stream/interface.go

```diff
@@ -0,0 +1,12 @@
+package stream
```

Comment:
Not sure if `stream` is the best name here, I know it's in line with what `go-streamprocessor` uses, but I think we should reflect that it's just a wrapper around Kafka specifically. What do you think?

Reply:
I called it `kafka` first, but that really conflicted with the rdkafka package.. I'm not happy with `stream` either.